OPHD/ophd.vcxproj

```diff
@@ -105,6 +105,16 @@
       <TargetMachine>MachineX86</TargetMachine>
       <AdditionalLibraryDirectories>$(SolutionDir)$(Configuration)\;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
+    <PostBuildEvent>
+      <Command>SET sourceDir="$(VcpkgCurrentInstalledDir)bin\"
+if $(ConfigurationName) == Debug (
+  SET sourceDir="$(VcpkgCurrentInstalledDir)debug\bin\"
+)
+
+xcopy /y /d "%sourceDir%ogg.dll" "$(OutDir)"
+xcopy /y /d "%sourceDir%vorbis.dll" "$(OutDir)"
+xcopy /y /d "%sourceDir%vorbisfile.dll" "$(OutDir)"</Command>

```

Comment:
Any idea what the difference is between `$(OutDir)` and `$(TargetDir)`?

Reference:
https://docs.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?redirectedfrom=MSDN&view=vs-2019

It's not clear to me which one should be used.

Reply:
No, I've been using OutDir successfully on other projects and as mentioned the documentation isn't too clear on what the difference is.

Reply:
I wonder if they might differ if `$(TargetPath)` is defined to be outside of `$(OutDir)`.

I noticed the Vcpkg script uses `$(TargetPath)` in its script, which is assume is more closely related to `$(TargetDir)` than to `$(OutDir)`. This may be splitting hairs though.